.github/workflows/format_codebase.yml

```diff
@@ -0,0 +1,46 @@
+name: 💅 Format Codebase
+
+on:
```

Comment:
Is the workflow triggered when a CSV or a JSON file is changed? Would we need to approve a PR every week if that's the case?

Would you consider changing the trigger to:
```json
on:
  push:
    paths-ignore:
      - '**.json'
      - '**.csv'
```

Reply:
No, the prettier would run automatically, only the black formatter (for python files) opens a PR.

Reply:
That's perfect!